app/Console/Commands/DevCommands.php

```
@@ -310,7 +309,6 @@ protected function newUserEmail()
 
     public function liveFlights(): void
     {
-        $acarsRepo = app(AcarsRepository::class);
-        $flights = $acarsRepo->getPositions(setting('acars.live_time'))->toArray();
+        $flights = Pirep::activeFlights(setting('acars.live_time'))->get()->toArray();
```

PHPMD (2.15.0)

[warning] Avoid unused local variables such as '$flights'. (undefined)

(UnusedLocalVariable)